Rocket.Chat/Models/Subscription/SubscriptionActions.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  SubscriptionActions.swift
+//  Rocket.Chat
+//
+//  Created by Samar Sunkaria on 8/16/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import RealmSwift
+
+extension Subscription {
+    func favoriteSubscription() {
+        SubscriptionManager.toggleFavorite(self) { (response) in
```

Comment:
These functions should be in `struct SubscriptionsClient` so we can use other APIs than `API.current()` and also be able to test with MockAPI.